src/Algebra/Graph/Labelled.hs

```diff
@@ -482,7 +490,12 @@ transpose = foldg empty vertex (fmap flip connect)
 -- emap g . emap h        == emap (g . h)
 -- @
 emap :: (e -> f) -> Graph e a -> Graph f a
-emap f = foldg Empty Vertex (Connect . f)
+emap f =
```

Comment:
I find this a bit awkward. Let's keep `emap f = foldg Empty Vertex (Connect . f)` for now (at least until we need to support pre-Bifunctor `base`).